client/stylesheets/_toCat.scss

```diff
@@ -2897,7 +2897,8 @@ i:hover {
 }
 
 .managerOfferView {
-    margin-top: 20px !important;
+    margin-top: 23px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin-bottom, margin-top, text-align